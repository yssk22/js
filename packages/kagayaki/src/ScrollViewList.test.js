/* @flow */
import React from 'react';
import { Text } from 'react-native';
import ScrollViewList from './ScrollViewList';
import renderer from 'react-test-renderer';

describe('ScrollViewList', () => {
  test('render', () => {
    const component = renderer.create(
      <ScrollViewList
        data={[{ a: 1 }]}
        childComponentFunc={(v, idx) => {
          return <Text key={idx}>{v.a}</Text>;
        }}
      />
    );
    expect(component.toJSON()).toMatchSnapshot();
  });
});
