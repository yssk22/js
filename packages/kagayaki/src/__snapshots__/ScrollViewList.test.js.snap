// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollViewList render 1`] = `
<RCTScrollView
  refreshControl={
    <RefreshControlMock
      refreshing={false}
    />
  }
>
  <RCTRefreshControl />
  <View>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      1
    </Text>
  </View>
</RCTScrollView>
`;
